contracts/test/TestTransfers.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+import "../base/Transfers.sol";
+import "../interfaces/ITransfers.sol";
+import 'hardhat/console.sol';
```

Comment:
get rid of reference to hardhat/console.sol

Reply:
alright